src/main/webapp/WEB-INF/jsp/help.jsp

```diff
@@ -59,9 +59,9 @@
             <div class="panel-body">
                 <img src="${context}/img/Setup.png">
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add an "alt" attribute to this image. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AImgWithoutAltCheck)